libcfnet/tls_client.c

```diff
@@ -35,6 +35,7 @@
 #include <tls_client.h>
 #include <tls_generic.h>
 #include <net.h>                     /* SendTransaction, ReceiveTransaction */
+#include <protocol.h>                      /* ParseProtocolVersionNetwork() */
```

Comment:
Looks like this should be a part of a different commit, but not big deal, of course.

Reply:
Yep, commit after, thanks, fixed.